docs/src/pages/components/stack/InteractiveStack.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Grid from '@material-ui/core/Grid';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
+import Paper from '@material-ui/core/Paper';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Stack, { StackProps } from '@material-ui/core/Stack';
+
+export default function InteractiveStack() {
+  const [direction, setDirection] = React.useState<StackProps['direction']>('row');
+  const [justifyContent, setJustifyContent] = React.useState('center');
+  const [alignItems, setAlignItems] = React.useState('center');
+  const [spacing, setSpacing] = React.useState(1);
+
+  const jsx = `
+<Stack
+  direction="${direction}"
+  justifyContent="${justifyContent}"
+  alignItems="${alignItems}"
+  spacing={${spacing}}
+>
+`;
+
+  return (
+    <Stack sx={{ flexGrow: 1 }}>
+      <Stack
+        direction={direction}
+        justifyContent={justifyContent}
+        alignItems={alignItems}
+        spacing={spacing}
+        sx={{ height: 240 }}
+      >
+        {[0, 1, 2].map((value) => (
+          <Paper
+            key={value}
+            sx={{
+              padding: 2,
+              paddingTop: value + 1,
+              paddingBottom: value + 1,
```

Comment:
We have always used the shorthand in the docs so far for the `sx`.

```suggestion
              p: 2,
              pt: value + 1,
              pb: value + 1,
```